### PR TITLE
macOS setup documentation

### PR DESCRIPTION
**Pull request by mrhelmut**
This is an attempt to address issues such as this one: https://github.com/MonoGame/MonoGame/issues/7982

### CODE REVIEW

File: content/articles/getting_started/1_setting_up_your_development_environment_unix.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ The only development environment that MonoGame officially supports on Linux is [
...
+- If you are intending to target mobile platforms, make sure that you have the corresponding workload installed
+
+```sh
+dotnet workload install ios
+dotnet workload install android
+```
+
+## macOS considerations
+
+For the time being, MonoGame requires that you install the x64 version of the .NET SDK even if you are running on an Apple Silicon mac. It is also required that [Rosetta](https://support.apple.com/en-us/HT211861) is enabled.
```
Comment (AristurtleDev) [inline content/articles/getting_started/1_setting_up_your_development_environment_unix.md:24]: Even with Rosetta emulation, the mgcb will fail to build Textures and SpriteFonts due to the freeimage and freetype libs needing to be built for the M1/M2 architecture.  

There are workarounds that some have used such as doing a brew install of both libs and creating a symnlink for them

**Review by AristurtleDev — COMMENTED** [review]
Include considerations about freeimage and freetype?

**Comment by mrhelmut** [comment]
> Include considerations about freeimage and freetype?

We should rebuild those libraries with Apple Silicon support, the setup on macOS and Linux is already convoluted enough. I'll try to setup that in the way @harry-cpp initiated with Github Actions for other libs.

**Comment by harry-cpp** [comment]
@AristurtleDev and I already set them up ^^

**Comment by harry-cpp** [comment]
The links in case you were wondering:
https://github.com/MonoGame/MonoGame.Library.FreeType
https://github.com/MonoGame/MonoGame.Library.FreeImage

**Comment by mrhelmut** [comment]
You're always a step ahead (or I'm lagging).

So all we need is updating the references? (and make sure that SharpFont can load FreeType properly)

**Comment by AristurtleDev** [comment]
Yea, the references packed with the mgcb NuGet need to be updated to include the the apple silicon built libs.  That still won't resolve *current* user issue though until a new MonoGame release is done, so the information about it should probably still be included in the macOS consideration section written for the page until then.

**Review by tomspilman — APPROVED** [review]

**Pull request merged**